build.gradle

```diff
@@ -87,7 +94,11 @@ publishing {
     publications {
         maven(MavenPublication) {
             groupId = 'org.opendatadiscovery'
-            artifactId = 'oddrn-generator-java'
+            if (spark_compatible) {
+                artifactId = 'oddrn-generator-java8'
+            } else {
+                artifactId = 'oddrn-generator-java'
```

Comment:
@dliubimov
should we make it 'oddrn-generator-java11' then?
what's the best practice for naming here?